scripts/generate_docs.sc

```diff
@@ -0,0 +1,101 @@
+#!/usr/bin/env amm
+
+import java.time.Instant
+import ammonite.ops._
+import ammonite.ops.ImplicitWd._
+
+/**
+  * Docs generation process:
+  *
+  * 1. copy docs/docs and docs/_layouts/docs.html part from 1.3, 1.4 and 1.5 branches to the temp dirs
+  * 2. copy the latest release (1.5_ branch doc to the temp dir
+  * 3. generate the latest release docs
+  * 4. replace docs/docs with docs/docs from 1.3, 1.4, 1.5 and generate docs for the _sites/docs_version
+  */
+
+def makeTmpDir(): Path = {
+  val timestamp = Instant.now().toString.replace(':', '-')
+  val path = root/"tmp"/s"marathon-docs-build-$timestamp"
+  mkdir! path
+  path
+}
+
+val latestReleaseVersion = "1.5"
+
+val marathonVersions = List("1.1", "1.3", "1.4", latestReleaseVersion)
+
+val buildDir = makeTmpDir()
+
+val docsDir = pwd/up/"docs"
+
+%ls docsDir
+
+%git('checkout, "master")
```

Comment:
Maybe I'm missing something here. Where and when is this run? Should the caller decide which commit is built?

Reply:
Good catch, this is a dead code